README.md

```diff
@@ -1,39 +1,46 @@
 [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
-
+[![PyPI version](https://badge.fury.io/py/pycrostates.svg)](https://badge.fury.io/py/pycrostates)
+[![Documentation Status](https://readthedocs.org/projects/pycrostates/badge/?version=latest)](https://pycrostates.readthedocs.io/en/latest/?badge=latest)
```

Comment:
```suggestion
```

Duplicated of badge below?

Reply:
I don't think this is not really an issue. Feel free to delete it  before merging if you think it's better